### PR TITLE
Update Persian_(Pinglish-UN).properties

### PR DESCRIPTION
**Pull request by Herbad**
Translate some English words

### CODE REVIEW

**Comment by yairm210** [comment]
Sorry, didn't notice that the tests never ran on these.

Errors:
```

    Placeholder `[count]` not found in `Persian_(Pinglish-UN)` for entry `We promised not to settle near them ([count] turns remaining)`
    Placeholder `[count]` not found in `Persian_(Pinglish-UN)` for entry `They promised not to settle near us ([count] turns remaining)`
    Placeholder `[amount]` not found in `Persian_(Pinglish-UN)` for entry `[amount] Buildings`
    Placeholder `[amount]` not found in `Persian_(Pinglish-UN)` for entry `[amount] Resources`
    Placeholder `[amount]` not found in `Persian_(Pinglish-UN)` for entry `[amount] Improvements`
    Placeholder `[amount]` not found in `Persian_(Pinglish-UN)` for entry `[amount] Religions`
    Placeholder `[amount]` not found in `Persian_(Pinglish-UN)` for entry `[amount] Beliefs`
    Placeholder `[saveFileName]` not found in `Persian_(Pinglish-UN)` for entry `[saveFileName] deleted successfully.`
```

**Comment by yairm210** [comment]
https://yairm210.github.io/Unciv/Other/Translating/#pitfalls

**Comment by Herbad** [comment]
> https://yairm210.github.io/Unciv/Other/Translating/#pitfalls

Sorry, it didn't ran? should I do now? Explain me please

File: android/assets/jsons/translations/Persian_(Pinglish-UN).properties [hunk 1]
```diff
@@ -598,22 +579,14 @@ You can play it, but don't expect everything to work! =
...
- # Requires translation!
-[amount] Beliefs = 
+[amount] Techs = [amount] Teknoloji ha
+[amount] Nations = [amount] Meliat ha
+[amount] Units = [amount] Niroo ha
+[amount] Buildings = Sakhteman ha
+[amount] Resources = Manaabe’e
+[amount] Improvements = Pishraft ha
+[amount] Religions = Adian
+[amount] Beliefs = Bavar ha
```
Comment (yairm210) [inline android/assets/jsons/translations/Persian_(Pinglish-UN).properties:589]: Half of these don't have the [amount] part they should have

**Comment by yairm210** [comment]
We check if the translations are OK before we add them to the project.
In this case the check didn't run on time so I had to revert the changes.
What you need to do is:
A. Fix the missing placeholders  - "[amount]"
B. Create the PR again

**Comment by Herbad** [comment]
> We check if the translations are OK before we add them to the project. In this case the check didn't run on time so I had to revert the changes. What you need to do is: A. Fix the missing placeholders - "[amount]" B. Create the PR again

I reverted something accidentally🚶🏽‍♂️

**Pull request merged**